### DISCUSSION TITLE
On the MRF utility in DASimpleFoam

### DISCUSSION THREAD

**Discussion by chairmanmao256** [mdolab/dafoam#293] [General]
Hi! I'm here again! I've been using DAFoam v2.2.9 since its release and all my modifications of the source code are based on v2.2.9. Now I want to use the MRF utility and I found the MRF-related stuff in the source code of DASimpleFoam. I also found MRF-stuff in `DAResidualSimpleFoam.C`, but the comment in this file says: `We don't support MRF and fvOptions so all the related lines are commented out for now.` 
So I want to ask if the MRF utility is supported in DASimpleFoam of v2.2.9 or do I have to install v2.2.10 to use the MRF utility? Thank you!

**Comment by friedenhe** [comment]
That should be an outdated comment, MRF and fvOptions have been added to some of the solvers, including DAsimpleFoam.

**Reply by chairmanmao256** [reply]
I've found that adding the inertial forces manually (SRF) is more suitable in my case. But it is also good to know MRF can function properly in v2.2.9, thank you!

**Reply by DUT-wansy** [reply]
Hello, Dalao.
I notice that you have already added SRF to DASimpleFoam, could you please give me some advise on how to implement it?

**Reply by chairmanmao256** [reply]
Hi, I've already e-mailed you about this issue! Hope it can help you.

**Reply by raisePiginYourhome** [reply]
can you share some experience about how to use SRF for turbomachinery optimization, thanks a lot
